CRM/Activity/Form/Task/PDF.php

```diff
@@ -31,18 +33,24 @@ public function preProcess() {
    */
   public function buildQuickForm() {
     $this->addPDFElementsToForm();
-    // Remove types other than pdf as they are not working (have never worked) and don't want fix
-    // for them to block pdf.
```

Comment:
So far so good just this seems like a useful comment? Otherwise it's not clear why this override is here.

Reply:
@demeritcowboy good catch - I had wondered why that line was there, not realising I had removed the explanation - back now